services/src/ogc/util.rs

```diff
@@ -347,27 +358,11 @@ mod tests {
         assert_eq!(
             parse_wcs_bbox(to_deserializer(s)).unwrap(),
             WcsBoundingbox {
-                partition: SpatialPartition2D::new_unchecked(
-                    (-162., 81.).into(),
-                    (162., -81.).into()
-                ),
-                spatial_reference: SpatialReference::new(SpatialReferenceAuthority::Epsg, 3857),
-            }
-        )
-    }
-
-    #[test]
-    fn it_parses_wcs_bbox_epsg_4326() {
```

Comment:
why remove a test?

Reply:
because the axis ordering is no longer done during parsing now, as the spatial reference is optional now. It has to be done in a separate stept and is tested there